provisioning_templates/snippet/ansible_provisioning_callback

```diff
@@ -0,0 +1,28 @@
+<%#
+kind: snippet
+name: ansible_provisioning_callback
+-%>
+<% if host_param_true?('ansible_tower_provisioning') -%>
+<%
+  rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
```

Comment:
We duplicate these a lot. I wonder if there's a better solution (though that's out of scope here).